readme.md

```diff
@@ -130,19 +130,22 @@ This will scan the default style assets directory *assets/styles/* of the curren
 
 ---
 
-### Add Crossorigin Attr
+### Add Attr
 
-    ->addCrossoriginAttr( ...args )
+    ->addAttr( ...args )
 
-Uses the [`script_loader_tag`](https://developer.wordpress.org/reference/hooks/script_loader_tag/) filter to add `crossorigin="anonymous"` attribute to a specific script. For more details on this attribute see the following [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes).
+Uses the [`script_loader_tag`](https://developer.wordpress.org/reference/hooks/script_loader_tag/) filter to add an attribute and a value to a specific script for example crossorigin="anonymous". 
+To add attributes such as `async` or `defer` which don't required to be set to a value just set those attributes in the `integrations.yml` file to `true`.
```

Comment:
"Uses the [`script_loader_tag`](https://developer.wordpress.org/reference/hooks/script_loader_tag/) filter to add an attribute and a value to a specific script. For example; crossorigin="anonymous".  Attribute values may be strings. To add a boolean attribute such as `async` or `defer` pass `true`."